stripe/res/layout/add_address_widget.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <android.support.v7.widget.AppCompatSpinner
+        android:id="@+id/spinner_country"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp" />
```

Comment:
General nit:
Prefer to put the end of an angle bracket closure on a line by itself:
```
<TextInputLayout
    android:id="@+id/some_id"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    />
```

The rationale is that if you need to add another modifier to the item in a later diff and it goes at the bottom, then you only have to change one line rather than two (the `\>` by itself won't register as changed, but if you move it from being after `android:layout_marginTop` to being after `android:somethingElse`, then you changed both lines)